packages/blockchain-wallet-v4-frontend/src/scenes/Register/SimpleBuyInfo/index.tsx

```diff
@@ -0,0 +1,132 @@
+import { CoinType } from 'core/types'
+import { Icon, Image, Text } from 'blockchain-info-components'
+import { prop, propOr } from 'ramda'
+import React from 'react'
+import styled from 'styled-components'
+
+const SimpleBuyItemWrapper = styled.div`
+  display: flex;
+  margin-bottom: 1rem;
+  width: 100%;
+`
+
+const SimpleWrapper = styled.div`
+  display: flex;
+`
+
+const AmountWrapper = styled.div`
+  display: flex;
+  border-radius: 0.5rem;
+  justify-content: space-between;
+  align-items: center;
+  flex: 1;
+  border: 1px solid ${props => props.theme.grey100};
+  padding: 0.75rem;
+  margin-right: 1rem;
+  margin-top: 2rem;
+`
+
+const CryptoWrapper = styled(AmountWrapper)`
+  flex: 3;
+  justify-content: initial;
+  margin-right: 0;
+
+  > * {
+    margin-right: 0.5rem;
+  }
+`
+
+type IconType =
+  | 'btc-circle-filled'
+  | 'eth-circle-filled'
+  | 'bch-circle-filled'
+  | 'xlm-circle-filled'
+  | 'usd-d'
+  | 'algo'
+  | 'usdt'
+  | 'btc-circle-filled'
+
+type ColorIconType =
+  | 'btc'
+  | 'eth'
+  | 'bch'
+  | 'xlm'
+  | 'pax'
+  | 'algo'
+  | 'usdt'
+  | 'usd-d'
+
+const getIcon = (crypto: CoinType): IconType => {
+  switch (crypto) {
+    case 'BTC':
+      return 'btc-circle-filled'
+    case 'ETH':
+      return 'eth-circle-filled'
+    case 'BCH':
+      return 'bch-circle-filled'
+    case 'XLM':
+      return 'xlm-circle-filled'
+    case 'PAX':
+      return 'usd-d'
+    case 'ALGO':
+      return 'algo'
+    case 'USDT':
+      return 'usdt'
+    default:
+      return 'btc-circle-filled'
+  }
+}
+
+const getColor = (crypto: CoinType): ColorIconType => {
+  switch (crypto) {
+    case 'BTC':
+      return 'btc'
+    case 'ETH':
+      return 'eth'
+    case 'BCH':
+      return 'bch'
+    case 'XLM':
+      return 'xlm'
+    case 'PAX':
+      return 'usd-d'
+    case 'ALGO':
+      return 'algo'
+    case 'USDT':
+      return 'usdt'
+    default:
+      return 'btc'
+  }
+}
+
+const SimpleBuyInfo = ({ goalData }) => (
+  <SimpleBuyItemWrapper>
+    <AmountWrapper>
+      <SimpleWrapper>
+        <Text size='16px' color='grey400' weight={500}>
+          $
+        </Text>
+        <Text size='16px' color='black' weight={500}>
+          {prop('amount', goalData)}
+        </Text>
+      </SimpleWrapper>
+      <Image name='us-flag' height='24px' />
+    </AmountWrapper>
+
+    <CryptoWrapper>
+      <Icon
+        color={getColor(propOr('BTC', 'crypto', goalData))}
+        name={getIcon(propOr('BTC', 'crypto', goalData))}
```

Comment:
Please use wallet options config to get icons and colors and remove both `getIcon` and `getColor` functions.  there are examples all over the codebase.  They are defined as such here => https://github.com/blockchain/wallet-options/blob/master/prod/wallet-options-v4.json#L89-L101